### DISCUSSION TITLE
Plugin Scan Failed

### DISCUSSION THREAD

**Discussion by WoodenCubeAudio** [apohl79/audiogridder#685] [Help] [answered]
Hey guys.
I absolutely love this plugin!

Unfortunately after trying it on a fresh installed Windows 10 Home Pc Audiogridder Server Tools shows just "failed" and all plugins marked red. Does anybody knows what to do?
All of them should be licenced.
Is it maybe some Windows configuration / restriction?
Scanning and using them in e.g. Reaper works fine. :(

Here are the .log files:
[AudioGridder log.zip](https://github.com/apohl79/audiogridder/files/6939408/AudioGridder.log.zip)

I've already tried to reinstall Windows 10 Home (clean version directly from Microsoft), clearing folder in "roaming" several times, installing 1.11 and 1.20 beta...
It works on my main DAW Pc and my laptop.

Pls help :)

**Comment by WoodenCubeAudio** [comment, marked as answer]
Think I've got the solution.
It seems like there is an internal issue with windows and C++ Visual Studio.
Maybe just for Windows 10 Home Edition?

Anyways, download those installers. Seems to work:

https://support.microsoft.com/de-de/topic/aktuelle-unterst%C3%BCtzte-downloads-f%C3%BCr-visual-c-2647da03-1eea-4433-9aff-95f26a218cc0

**Reply by TitannX** [reply]
Dude you saved me

**Reply by rub3nItaly** [reply]
thanks!! You solved my problem ✌️

**Reply by nsputnik** [reply]
I am trying to just get one plugin to work and it says the plugin failed, Windows 10, x64.  I did try installing Microsoft Visual C++.  The plugin runs fine in Nanohost and Savihost.
